.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,20 +15,20 @@ assignees: ''
 #### Steps to reproduce
 
 ##### `crash.log` (if applicable)
-`crash.log` is a file that is automatically created when KOReader crashes. It can
-normally be found in the KOReader directory:
+`crash.log` is a file that is automatically created when KOReader crashes. It can normally be found in the KOReader directory:
 
 * `/mnt/private/koreader` for Cervantes
 * `koreader/` directory for Kindle
 * `.adds/koreader/` directory for Kobo
 * `applications/koreader/` directory for Pocketbook
 
-
-Android logs are kept in memory. Please go to Help → Bug Report to save these logs to a file.
+Android logs are kept in memory. Please go to [Menu] → Help → Bug Report to save these logs to a file.
 
 Please try to include the relevant sections in your issue description.
-You can upload the whole `crash.log` file on GitHub by dragging and
-dropping it onto this textbox.
+You can upload the whole `crash.log` file on GitHub by dragging and dropping it onto this textbox.
```

Comment:
Also zipped if necessary? Not sure what the filesize limit is otoh (10 MB?) but zipping does cut it very drastically.